packages/renderer/src/override.css

```diff
@@ -16,6 +16,8 @@
   --pf-global--LineHeight--md: 0.9rem;
   --pf-global--FontWeight--normal: 100;
   --pf-global--BackgroundColor: #ff0000;
+
+  --bs-primary: #8c4afd;
```

Comment:
hello, I would expect to see a color coming from our 'palette color' in tailwind.config.cjs but I don't see a matching value for `#8c4afd` in our palette.

here it looks like you're using one of the patternfly colors
```
  --pf-global--primary-color--300: #8c4afd;
  --pf-global--primary-color--light-300: #8c4afd;
  --pf-global--primary-color--dark-300: #8c4afd;
```

1. I would like to reuse a color from tailwind palette and if possible to not duplicate the value and reuse the color definition

so to use `--bs-primary: theme(colors.purple.500);` (if we want the 500 color from purple theme)


2. Last, as the Steps definition is the same across all the Provider

maybe we would need a <ProviderStep> that do all the settings like `size="1.7rem" line="1px" current="{0}" clickable="{false}" />`


and maybe, we could have a `<Steps> component in the dashboard that does the customization of the color, CSS locally and not for the full body) so in the app we import from our 'Steps' and not from the library
Like what we did for `<Route`

but here it can be follow-ups PRs

but we really need to reuse the definition and not duplicate the color definition